client/assets/js/main.js

```diff
@@ -1,74 +1,85 @@
-$(document).ready(init);
-
+document.addEventListener("DOMContentLoaded", init);
 
 function init(){
   handleClickEvents();
 }
 
 function handleClickEvents(){
-  $('.open').on('click', function(){ //go to stories to check, leaves landing page
-    $('#landing-container').addClass('ghostOut').fadeOut(3000);
+  const greeting = document.querySelector('.greeting');
+  const landingContainer = document.getElementById('#landing-container');
+  const open = document.querySelector('.open');
+  const signUp = document.querySelector('.signUp');
+  const logIn = document.querySelector('.logIn');
+  const formLog = document.querySelector('.formLog');
+  const formSign = document.querySelector('.formSign');
+  const instruction = document.querySelector('.instruction');
+  const sign = document.querySelector('.sign');
+
+  open.onclick = function(){ //go to stories to check, leaves landing page
+    landingContainer.classList.add('ghostOut').classList.add('fade-out-transition');
     setTimeout(function(){
-      $('#landing-container').removeClass('ghostOut')
+      landingContainer.classList.remove('ghostOut');
     },3100)
-  });
+  };
 
-  $('.experience').on('click', function(){ //go to map and camera
+  document.querySelector('.experience').onclick = function(){ //go to map and camera
     if(window.outerWidth < 750 ){
       window.open('meter-index.html','_self')
     }else{
-      $('.instruction').fadeIn();
+      instruction.classList.add('fade-in');
     }
-  });
-
-  $('.story').click(function(){ //shows the story summary
-    $('.caption', this).slideDown('linear');
-  });
-
-  $('.story').mouseleave(function(){//hidesthe story summary
-    $('.caption',this).slideUp('linear');
-  });
+  };
 
-  $('.signUp').on('click',function(){ //clicking the sign up button at the landing page
-    $('.greeting').fadeOut();
-    $('.formSign').fadeIn();
-  });
+  document.querySelector('.signUp').onclick = function(){ //clicking the sign up button at the landing page
+    greeting.classList.add('fade-out');
+    formSign.classList.add('fade-in');
+    sign.classList.remove('fade-out');
+    sign.classList.remove('ghostOut');
+  };
 
-  $('.logIn').on('click',function(){//clicking the log in button at the landing page
-    $('.greeting').fadeOut();
-    $('.formLog').fadeIn()
-  });
+  document.querySelector('.logIn').onclick = function(){//clicking the log in button at the landing page
+    greeting.classList.add('fade-out');
+    formLog.classList.add('fade-in');
+  };
 
-  $('.cancel').on('click',function(){ // clicking the cancel on the sign up and log in forms
-    $(this).parent().parent().addClass('ghostOut').fadeOut(2500)
-    setTimeout(function(){
-        $('.formSign, .formLog').removeClass('ghostOut')
-    },3100)
-    $('.greeting').fadeIn();
-  });
+  document.querySelector('.cancel').onclick = function (event) {
+    const parent = event.target.parentNode;
+    parent.classList.add('ghostOut');
+    parent.classList.add('fade-out');
```

Comment:
the ghost out isn't working properly and is overwritten by` fade-out.` So you don't see the animation at all. If you're having problems just stick with `fade-out`.